http/src/client/mod.rs

```diff
@@ -2306,6 +2324,18 @@ impl Client {
     /// println!("{sticker:#?}");
     /// # Ok(()) }
     /// ```
+    ///
+    /// # Errors
+    ///
+    /// Returns an error of type [`DescriptionInvalid`] if the length is invalid.
+    ///
+    /// Returns an error of type [`NameInvalid`] if the length is invalid.
+    ///
+    /// Returns an error of type [`TagsInvalid`] if the length is invalid.
```

Comment:
Potential followup: Maybe we could agree on a some style for error variants depending on how many there are. Something like a bullet list is be easier to read. In this case the sentence itself is not very useful as it doesn't tell you which field returns what variant, you have to infer it yourself based off the error variant name.